ceremonies/src/lib.rs

```diff
@@ -428,6 +440,19 @@ pub mod pallet {
 			}
 
 			<MeetupTimeOffset<T>>::put(meetup_time_offset);
+			info!(target: LOG, "set meetup time offset to {} ms", meetup_time_offset);
+			Self::deposit_event(Event::MeetupTimeOffsetUpdated(meetup_time_offset));
+			Ok(().into())
+		}
+		#[pallet::weight((1000, DispatchClass::Operational,))]
+		pub fn purge_ceremony_history(
+			origin: OriginFor<T>,
+			community_ceremony: CommunityCeremony,
+		) -> DispatchResultWithPostInfo {
+			<T as pallet::Config>::CeremonyMaster::ensure_origin(origin)?;
+
+			Self::purge_community_ceremony(community_ceremony);
+
 			Ok(().into())
 		}
```

Comment:
I'd rename the call to `purge_community_ceremony` the current name does not reflect the behaviour.